### ISSUE TITLE
Some UI elements get stretched at ultrawide resolutions

### ISSUE THREAD

**Issue by ReneSchimmelpfennig** [GTAmodding/GTAIV-Issues-List#100]
FusionFix has fixed the stretched cellphone and loading screens, the TBoGT mission completion screen, the police computer and in-game internet remain. :)

<img width="3440" height="1440" alt="Image" src="https://github.com/user-attachments/assets/318694d0-2ff8-4f37-b2bf-9e73f497bef5" />

![16 1](https://github.com/GTAmodding/GTAIV-Issues-List/assets/97738487/cf42a085-e007-48da-b6d8-f9da70e7ce39)
![21 1](https://github.com/GTAmodding/GTAIV-Issues-List/assets/97738487/6dce0714-9519-489c-b7d0-626118039b11)
![16 2](https://github.com/GTAmodding/GTAIV-Issues-List/assets/97738487/05b92fc5-5906-4f18-ba7e-19ff8288f3a4)
![21 2](https://github.com/GTAmodding/GTAIV-Issues-List/assets/97738487/b71a5845-c5d0-497f-a72f-d0d6cdd74761)

**Comment by EQQx110** [comment]
Actually, the elements that stretch start stretching at any resolution higher than 1024x768, with most elements having the proper aspect ratio at 1024x768 (or lower) (like the web, police computer background and keyboard) and some other at higher resolutions (like the mugshots and the police coat of arms in the police computer), of course none of these have the proper aspect ratio at ultrawide resolutions.

**Comment by Tomasak** [comment]
Websites are pillarboxed on 16:9 and police computer seems to fill in car model screen, can you post comparison pics?

**Comment by EQQx110** [comment]
> Websites are pillarboxed on 16:9

Yes but past 1024x768 they're stretched, wether it is in 4:3 or 16:9 doesn't matter. (I suppose the game zooms out/shrinks to better use the bigger resolution, but it only zooms out/shrinks vertically, which makes it stretched)

800x600:
<img width="800" height="600" alt="Image" src="https://github.com/user-attachments/assets/aab79a8b-1af6-4cc0-b5a7-4c4a7b416c7e" />

1024x768 (it looks the same as 800x600):
<img width="1024" height="768" alt="Image" src="https://github.com/user-attachments/assets/303cb89a-5c00-46e7-a56c-e28a27387809" />

1128x634 (is lower than 1024x768 so isn't stretched):
<img width="1128" height="634" alt="Image" src="https://github.com/user-attachments/assets/f67c3ff8-ecc5-4eb5-a135-08171b15a585" />

1440x1080:
<img width="1440" height="1080" alt="Image" src="https://github.com/user-attachments/assets/86e484a1-af42-4dc2-8957-9cec95b0a9ab" />

1920x1080:
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/8a89f8b0-2ed8-4f7f-84a5-6baed7c83d8d" />


1024x768:
<img width="1024" height="768" alt="Image" src="https://github.com/user-attachments/assets/be32035f-0735-4a0a-b97d-83ac028a5d4f" />

1128x634:
<img width="1128" height="634" alt="Image" src="https://github.com/user-attachments/assets/4437cf44-96e3-4920-a3d1-184891627a48" />

1440x1080:
<img width="1440" height="1080" alt="Image" src="https://github.com/user-attachments/assets/9937e616-e86d-40b4-8e05-9069af2001fc" />

1920x1080:
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/cfd45376-8d99-4968-951e-517b0a24be6d" />



Here are the images with the same aspect ratio next to eachother so it's easier to see the differences:

1024x768 next to 1440x1080
<img width="2048" height="768" alt="Image" src="https://github.com/user-attachments/assets/aa6e1129-6b21-42d0-9093-7fb0ea446e0e" />

1128x634 next to 1920x1080
<img width="2256" height="634" alt="Image" src="https://github.com/user-attachments/assets/2674165f-3b63-4ea4-b9ed-a712daaa6c22" />


1024x768 next to 1440x1080
<img width="2048" height="768" alt="Image" src="https://github.com/user-attachments/assets/41f5d6d1-3785-4e23-80d2-299ce351c4e3" />

1128x634 next to 1920x1080
<img width="2256" height="634" alt="Image" src="https://github.com/user-attachments/assets/6e27fd78-d155-4d04-a10a-8c751fcd7e2a" />

Here the mouse cursor seems to be the only element that stretches depending on the aspect ratio instead of the resolution.

Meanwhile the police computer only depends on the aspect ratio, just like the mouse cursor. The correct one seems to be 16:10, at least in most part, it might be a mix of differenct aspect ratios for different elements.

4:3 (1440x1080):
<img width="1440" height="1080" alt="Image" src="https://github.com/user-attachments/assets/86ccf668-6a6d-4429-9346-e8cc39cdb42c" />

16:10 (1440x900):
<img width="1440" height="900" alt="Image" src="https://github.com/user-attachments/assets/a000e726-973c-4b98-9fd6-649050badfde" />

16:9 (1920x1080):
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/b8d49fc7-36dc-42f2-b7fc-b83492d20fe3" />


4:3 (1440x1080):
<img width="1440" height="1080" alt="Image" src="https://github.com/user-attachments/assets/6bea621c-b3c5-4c92-97aa-dc1eccb78674" />

16:10 (1440x900):
<img width="1440" height="900" alt="Image" src="https://github.com/user-attachments/assets/f16f9efd-2016-4fb6-883d-6b9b251118b0" />

16:9 (1920x1080):
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/216a9f38-7f7f-4de3-9bdb-254ae01e13aa" />
